authres_test.go

```diff
@@ -44,6 +44,19 @@ func TestParseAuthenticationResults(t *testing.T) {
 				},
 			},
 		},
+		{
+			`example.com; spf=neutral smtp.mailfrom=""`,
```

Comment:
Don't you want to add a test case where an empty value was passed without quotes? It must fail, doesn't it?

Reply:
yes it does, but I was planning to do it after the next pull request where we have some mechanism of collecting errors and then compare against that.